src/cross_section/src/cross_section.cpp

```diff
@@ -305,6 +305,24 @@ CrossSection CrossSection::Transform(const glm::mat3x2& m) const {
   return transformed;
 }
 
+CrossSection CrossSection::Warp(
```

Comment:
Also, either now or in a follow-on PR, would you mind adding docs to these functions? The CrossSection [docs](https://manifoldcad.org/docs/html/classmanifold_1_1_cross_section.html) are looking a little sparse.

Reply:
Sure, I'll make that my next PR.